src/components/WorkSchedule/WorkSchedule.js

```diff
@@ -29,6 +29,19 @@ class WorkSchedule extends Component {
     return "week" + Math.floor(Math.random() * 10);
   }
 
+  getFiveRandomRecipes(){
```

Comment:
it's give you five recipes or ingredients?

Reply:
I need rename on ingredients